### ISSUE TITLE
Error while executing the `rebuild` command

### ISSUE THREAD

**Issue by JoCat** [cmake-js/cmake-js#317]
I noticed a problem when installing the `webview-nodejs` package version `0.4.0`. It depends on the `libwebview-nodejs` package, which in turn depends on your `7.2.1` package. And during installation it tries to call the command `C:Windows\Windows\system32\cmd.exe /d /s /c cmake-js rebuild`, but when it is called the following error occurs:

```
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c cmake-js rebuild
npm ERR! F:\test\node_modules\cmake-js\lib\import\find-visualstudio.js:345
npm ERR!     if (this.nodeSemver.major >= 19) {
npm ERR!                         ^
npm ERR!
npm ERR! TypeError: Cannot read properties of undefined (reading 'major')
npm ERR!     at VisualStudioFinder.findVisualStudio2015 (F:\test\node_modules\cmake-js\lib\import\find-visualstudio.js:345:25)
npm ERR!     at F:\test\node_modules\cmake-js\lib\import\find-visualstudio.js:66:12
npm ERR!     at VisualStudioFinder.parseData (F:\test\node_modules\cmake-js\lib\import\find-visualstudio.js:232:5)
npm ERR!     at F:\test\node_modules\cmake-js\lib\import\find-visualstudio.js:138:14
npm ERR!     at ChildProcess.exithandler (node:child_process:414:7)
npm ERR!     at ChildProcess.emit (node:events:514:28)
npm ERR!     at maybeClose (node:internal/child_process:1105:16)
npm ERR!     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
npm ERR!
npm ERR! Node.js v20.9.0
```

Judging by the commit https://github.com/cmake-js/cmake-js/commit/14525bf4750824b2f87ef515a5d19b5b3249b267, the code from [node-gyp](https://github.com/nodejs/node-gyp) repository was copied thoughtlessly, as evidenced by the absence of the `nodeSemver` variable.

**Comment by tangshuang** [comment]
facing the same problem when run on windows with cmake, vs, MinGW-w64, nodejs20 installed

**Comment by Julusian** [comment]
Fix has been released as part of v7.3.0
